web/src/components/core/SectionSkeleton.jsx

```diff
@@ -23,19 +23,27 @@ import React from "react";
 import { Skeleton } from "@patternfly/react-core";
 import { _ } from "~/i18n";
 
-const SectionSkeleton = () => (
-  <>
-    <Skeleton
-      screenreaderText={_("Waiting")}
-      fontSize="sm"
-      width="50%"
-    />
-    <Skeleton
-      screenreaderText={_("Waiting")}
-      fontSize="sm"
-      width="25%"
-    />
-  </>
-);
+const SectionSkeleton = ({ numRows = 2 }) => {
```

Comment:
What about taking the opportunity for renaming the prop to just `rows`?

Reply:
Not sure. I would expect `rows` to be a collection of objects defining each row (e.g., size of the row, etc).